modules/migration-kubernetes-objects.adoc

```diff
@@ -44,3 +51,19 @@ spec:
 ----
 <1> Specify the Kubernetes object, for example, `Secret` or `ConfigMap`.
 <2> Specify the label of the resources to migrate, for example, `app: frontend`.
+
+. Create a `MigMigration` CR to migrate the selected Kubernetes resources. Verify that the correct `MigPlan` is referenced in `migPlanRef`:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigMigration
+metadata:
+  generateName: <migplan>
+  namespace: openshift-migration
+spec:
+  migPlanRef:
+    name: <migplan>
+    namespace: openshift-migration
+  stage: false
+----
```

Comment:
@RichardHoch 
This module looks a bit confusing to me. You say "You add Kubernetes objects to the `MigPlan` CR by using **one of the following options**" and list three options. Then a single procedure follows. Which of the options does it correspond to? Also, the third option looks like a combination of the first and the second.

Reply:
I don't understand the confusion. The main addition of the PR is adding step 2 to the procedure. The three options were there before; the user chose which one option to use in step 1, which used to be the entire procedure. Same now,]. All i have added, besides step 2, is a bit more explanation to the three existing options. 
I changed to lead-in to read "one or more of the following options" but that's probably incorrect. Does that change reduce any confusion?

Reply:
@RichardHoch
Both `includedResources` and `labelSelector` appear in the code - so the procedure illustrates the third option, which is a combination of option 1 and option 2, correct? Maybe you should somehow indicate that. 
By the way, what does option 3 add as compared to option 2? Why would you want first to add the objects to be migrated as a list in `includedResources` and then filter them with `labelSelector`, when you can just use `labelSelector`? But maybe it's obvious to the reader.

Reply:
I agree that structure of this module is atypical. The description is a little larger than you'd see in most procedures, and it contains information that could be contained within the procedure itself. I also think that it would be defensible to break it up into two procedure modules, as you do essentially describe two procedures to accomplish the same goal in it. Less dramatically, you could structure the procedure as a conditional ordered list.